buildSrc/build.gradle

```diff
@@ -78,6 +83,7 @@ task writeVersionProperties {
 processResources {
   dependsOn writeVersionProperties
   from tempPropertiesFile
+  from wrapperProperties
```

Comment:
I think we should rename this file so it is more obvious what we are using it for. Or maybe not write the entire file, just the version.

Reply:
I initially added the file as is thinking about parsing it from the URL, but then realized that the wrapper know about the version it's going to use. In retrospect I'm not sure why I choose to add it to the properties files, I think the initial idea rooted, it makes much more sense to add a `minimumGradleVersion` resource that just has this version number.